common/changes/@reshuffle/db/docs_2019-09-19-14-47.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@reshuffle/db",
+      "comment": "Generate documentation using typedoc",
+      "type": "patch"
```

Comment:
Can be a `none` type since the package is unchanged, the documentation does not become a part of the package.